apps/backend/app/agent/providers/llama_index.py

```
@@ -0,0 +1,111 @@
+        provider: str = settings.EMBEDDING_PROVIDER):
+
+        self._model = embedding_model
+        self._provider = provider
+        self._api_key = api_key
+        self._api_base_url = api_base_url
+        provider_obj = _get_real_provider(provider)
+        assert issubclass(provider_obj, BaseEmbedding), "Embedding provider must be e.g. a llama_index.embeddings.* class"
+        # Again, this doesn't work on 100% of the LlamaIndex embedding
+        # integrations, but it's a fairly reliable pattern, and works
+        # for the important ones such as OpenAILike.
+        kwargs_for_provider = {'model':model_name,
```

Ruff (0.12.2)

Undefined name `model_name`

(F821)